### ISSUE TITLE
Creating an Ethernet Network with VLAN ID 1 for HPE Synergy results in "VLAN ID is reserved and not allowed"

### ISSUE THREAD

**Issue by ChrisLynchHPE** [HewlettPackard/POSH-HPOneView#306]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
HPE OneView for Synergy Composer relaxed VLAN ID 1 restriction, and is no longer reserved.

### Actual Behavior
Executing `New-HPOVNetwork` with -VlanId set to 1 will result in an exception stating the VLAN ID is reserved and not allowed.

### Steps to reproduce

```PowerShell
[PS] C:\Users\clynch> New-HPOVNetwork -Name "Management Network (VLAN1)" -VlanId 1 -Purpose Management
New-HPOVNetwork : The VLAN ID 1 is reserved and cannot be used.  Please provide a different VLAN ID to the Network
resource you are creating.
At line:1 char:1
+ New-HPOVNetwork -Name "Management Network (VLAN1)" -VlanId 1 -Purpose ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (VlanID:String) [New-HPOVNetwork], NetworkResourceException
    + FullyQualifiedErrorId : ReservedVlanID,New-HPOVNetwork
```

**Comment by ChrisLynchHPE** [comment]
This is a validation error within the Cmdlet, and will be resolved in the next update.

**Comment by ChrisLynchHPE** [comment]
This was addressed in the [Release 4.00.1587.3400 ](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.00.1587.3400 ).

Closing.
